npm/private/utils.bzl

```diff
@@ -269,7 +269,7 @@ def _npm_registry_download_url(package, version, registries, default_registry):
     )
 
 def _is_git_repository_url(url):
-    return url.startswith("git+ssh://") or url.startswith("git+https://")
+    return url.startswith("git+ssh://") or url.startswith("git+https://") or url.startswith("git@")
```

Comment:
One liner fix. The rest of the PR is adding proper test coverage for git+ssh with a private github repo